boards/arm/verdin_imx8mp_m7/doc/index.rst

```diff
@@ -0,0 +1,290 @@
+.. _verdin_imx8mp_m7:
+
+Toradex Verdin iMX8M Plus SoM
+
```

Comment:
FWIW, this improperly formatted heading is causing the page to not show up in the documentation. This, plus the fact that the "build hello world" instructions were basically broken (wrong board name) makes me think that the documentation for this page could have use a more thorough review (at a minimum testing the page properly shows up in the documentation ;-))

Just spent 30 minutes cleaning things up in #65240, please have a look so that we can get the table of contents for Arm boards back to a better state :)

![image](https://github.com/zephyrproject-rtos/zephyr/assets/128251/510ab18a-75df-4636-bb31-d75ab029450e)

Reply:
Thanks for pointing this out, and sorry for missing it initially.

@GabrielHAFs can you take a look at this?

Reply:
I obviously linked to the wrong PR -- #65420